### ISSUE TITLE
Even deleting a warehouse took 45s

### ISSUE THREAD

**Issue by tom10271** [frappe/frappe_docker#71]
I am using ERPNext on my MacBook Air with Intel i5 5250U and 8GB RAM.

![Screen Shot 2019-07-04 at 9 44 05 AM](https://user-images.githubusercontent.com/5089629/60634145-73d5d580-9e40-11e9-8131-b976ae52774c.png)

I was deleting a warehouse and took 45s, is there any recommendation on optimizing speed? Odoo works much faster, I suspect it is the cause of Docker?

**Comment by chabad360** [comment]
Please do not open another issue, close this, and post the same information on the previous [issue](#70) and reopen it. I will respond there.

**Comment by tom10271** [comment]
Who are you? Just a contributor right? I honestly think you are honestly polite. This is not the same issue  I don't care if you don't reply at all. I hope what is wrong in your life turns good so that you don't need to be like this anymore. God bless you, if he can

**Comment by chabad360** [comment]
Oh! I'm so sorry, I must have gotten you confused with another issue. it's rather late at night, so I will look into this tomorrow morning. Have a good night :)

P.s. I am currently responsible for a large amount of the code that is in this repo...

**Comment by chabad360** [comment]
Ok, let me clarify, I got confused (due to the time at night) with a question that I was answering on the [erpnext fourms](https://discuss.erpnext.com), and I thought (from the way the title was phrased) that is was a response to this question (which I had confused to be on github), so I wasn't going start answering on another issue, sorry about that....

So currently the docker image has certain issues that -to be honest- I have no idea why it happens. It seems to have been solved by running bench in production mode... So maybe it's caused by `bench watch`. But, I'm working on merging that in, so hopefully I'll have a PR ready soon.
